src/main/features/claim/routes/confirmation.ts

```diff
@@ -3,12 +3,14 @@ import { Paths } from 'claim/paths'
 
 import ClaimStoreClient from 'claims/claimStoreClient'
 import Claim from 'claims/models/claim'
+import OwnershipChecks from 'app/auth/ownershipChecks'
```

Comment:
@geek-soft didn't we decide we don't use `default exports` in TypeScript?

Reply:
Why is this not a guard?

Reply:
You are right @yarpo , we tend to not use defaults. Please change it @dharmendrak